variables-skyline/index340.html

```diff
@@ -161,12 +161,12 @@
 </html>
 
 <!--
-  Gradients in CSS are a way to transition between colors across the **distance** of an element. They are applied to the `background-image` property. The sytax looks like this:
+  Gradients in CSS are a way to transition between colors across the **distance** of an element. They are applied to the `background` property. The syntax looks like this:
```

Comment:
Since this is the first gradient example, you could show how to apply them to a property.